### PR TITLE
feat: implement lazy loading for some rango-client dependencies

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

This PR implements lazy loading for certain dependencies within the rango-client to optimize the bundle size and improve application performance. By dynamically importing modules only when needed, we can reduce the initial load time and enhance the overall user experience, especially for users who may not require all features immediately.

Fixes # (issue)
The difference in bundle size before and after:

before

Screenshot from 2024-08-27 14-34-28

after

Screenshot from 2024-08-27 14-58-26

# How did you test this change?
 
 Because most of the changes have happened in Cosmos and Solana and Terra wallets and Signers, please test the wallets and Signers.

Packages that are lazily loaded: 
- @terra-money/terra.js
- @cosmjs/launchpad
- @cosmjs/stargate
- @keplr-wallet/cosmos
- @solana/web3.js
- cosmos-wallet
- @keplr-wallet/simple-fetch

# Checklist:

- [x] I have performed a self-review of my code

### CODE REVIEW

File: signers/signer-cosmos/src/helpers.ts [hunk 1]
```diff
@@ -133,7 +136,7 @@ export const executeCosmosTransaction = async (
   }
 };
 
-export function getsignedTx(
+export async function getsignedTx(
```
Comment (RyukTheCoder) [inline signers/signer-cosmos/src/helpers.ts:139]: `getsignedTx` now returns a promise so it should be used with `await` in `wallets/provider-walletconnect-2/src/signers/cosmos.ts`

**Review by RyukTheCoder — CHANGES_REQUESTED** [review]
Tnx, please check `signer-terra` as well. Also, it is better to document list of packages which has made lazy and the amount of reduction in bundle size.

File: signers/signer-cosmos/src/helpers.ts (hunk 1 above)
Comment (mikasackermn) [reply signers/signer-cosmos/src/helpers.ts:139]: Fixed it

**Review by RyukTheCoder — APPROVED** [review]
LGTM

**Comment by RyukTheCoder** [comment]
Blocked by [this](https://github.com/rango-exchange/rango-client/pull/861)

**Comment by RanGojo** [comment]
Closed in favor of https://github.com/rango-exchange/rango-client/pull/861

**Pull request closed without merging**